crypto/ec/ec_ameth.c

```diff
@@ -794,3 +794,199 @@ int ECParameters_print(BIO *bp, const EC_KEY *x)
 {
     return do_EC_KEY_print(bp, x, 4, EC_KEY_PRINT_PARAM);
 }
+
+static int print_bin(BIO *fp, const char *str, const unsigned char *num,
+                     size_t len, int off);
+
+int ECPKParameters_print(BIO *bp, const EC_GROUP *x, int off)
```

Comment:
Why on earth is this function moved?

Reply:
It's written in the commit message. To be able to not use `#include "internal/deprecated.h" in eck_prn.c`.

Reply:
Yeah, but source wise, that's quite confusing, with a printing function that's *not* in the source file you'd expect to find it in.

Reply:
I simply don't see that one removed inclusion is good enough reason.  Maybe the reason is under-explained?

Reply:
If I understand it correctly the inclusion of `"internal/deprecated.h"` means that the functions are built into the object file even in no-deprecated builds. But yeah, I do not insist on this commit being included. I am going to drop it.